bin/www

```diff
@@ -0,0 +1,40 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
```

Comment:
**The purpose of this file is explained in the Pull Request description**. Also, this could help us to configure the Express server depending on the environment.